lib/honeybadger/utils.ex

```diff
@@ -14,4 +14,17 @@ defmodule Honeybadger.Utils do
     |> tl
     |> Enum.join(".")
   end
+
+  @doc """
+  Resolve environment name from Application configuration or from Mix.env
+  """
+  def environment_name do
+    case Application.get_env(:honeybadger, :mix_env, nil) do
+      nil ->
+        Mix.env
```

Comment:
Yes we've had a problem with that in the past. That was the original cause of #8. If you build a release Mix gets compiled out.